mail-client/alot/alot-0.6.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 vcs-snapshot
+
+DESCRIPTION="Experimental terminal UI for net-mail/notmuch written in Python"
+HOMEPAGE="https://github.com/pazz/alot"
+SRC_URI="${HOMEPAGE}/tarball/${PV} -> ${P}.tar.gz"
```

Comment:
don't do this, just put the github link back in

Reply:
Ok fixed in the next upgrade

Reply:
the idea is not to reuse `$HOMEPAGE` inside `SRC_URI`, not to completely delete `HOMEPAGE`

Reply:
Ok I'll ugpgrade that but just for my information, since you are duplicating the URL what is the reason behind that choice ?